daemon/clone-mgr.c

```diff
@@ -1236,7 +1264,12 @@ seaf_clone_manager_add_task (SeafCloneManager *mgr,
                            email, more_info,
                            sync_wt_name,
                            error);
-    g_free (worktree);
+
+out:
+    if (worktree)
```

Comment:
You don't need to check whether a variable is NULL before g_free(). g_free() will check it.